clippy_lints/src/methods/or_fun_call.rs

```diff
@@ -86,6 +86,8 @@ pub(super) fn check<'tcx>(
         span: Span,
         // None if lambda is required
         fun_span: Option<Span>,
+        // `Some` if fn has second argument
+        rest_arg: Option<&hir::Expr<'_>>,
```

Comment:
NIT: I would move this up to the other argument parameters. I would also name this `second_arg`.

Reply:
Yes, it makes sense.
In `or_fun_call`, I'll take `second_arg` because we do not expect to target a function with more than three arguments.